theano/gpuarray/extra_ops.py

```diff
@@ -40,6 +41,7 @@ def get_params(self, node):
     def make_node(self, x):
         assert x.type.dtype == 'float32', "Only float32 supported for GpuCumSum"
 
+        self.context_name = infer_context_name(x)
```

Comment:
Do not assign to `self.context_name` here, this is wrong.
You can use a local variable like `context_name` if you want.